### ISSUE TITLE
NPE in 2.14 caused by {}

### ISSUE THREAD

**Issue by vasilyt** [jenkinsci/slack-plugin#498]
Gitlab commit with title _APM-7701 Removed unused '/analytics/projects/{projectId}/applications'_ causes NPE in Slack plugin 2.14 on Jenkins 2.150.2:

```ERROR: Build step failed with exception
java.lang.NullPointerException
	at java.util.regex.Matcher.getTextLength(Matcher.java:1283)
	at java.util.regex.Matcher.reset(Matcher.java:309)
	at java.util.regex.Matcher.<init>(Matcher.java:229)
	at java.util.regex.Pattern.matcher(Pattern.java:1093)
	at jenkins.plugins.slack.ActiveNotifier$MessageBuilder.extractReplaceLinks(ActiveNotifier.java:508)
	at jenkins.plugins.slack.ActiveNotifier$MessageBuilder.escape(ActiveNotifier.java:526)
	at jenkins.plugins.slack.ActiveNotifier$MessageBuilder.append(ActiveNotifier.java:382)
	at jenkins.plugins.slack.ActiveNotifier.getCommitList(ActiveNotifier.java:253)
	at jenkins.plugins.slack.ActiveNotifier.completed(ActiveNotifier.java:148)
	at jenkins.plugins.slack.SlackNotifier.perform(SlackNotifier.java:452)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:744)
	at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:690)
	at hudson.model.Build$BuildExecution.cleanUp(Build.java:196)
	at hudson.model.Run.execute(Run.java:1857)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:429)
```

Any commit message with _{someText}_ symbols in it will cause this NPE.

**Comment by vasilyt** [comment]
Will probably be fixed by https://github.com/jenkinsci/slack-plugin/pull/497

**Comment by timja** [comment]
Released as 2.15, may take some time for it to be available in the update centre
